src/OrchardCore.Modules/OrchardCore.Queries/Sql/Startup.cs

```diff
@@ -18,11 +20,13 @@ public override void ConfigureServices(IServiceCollection services)
         {
             services.AddScoped<IPermissionProvider, Permissions>();
             services.AddScoped<IDisplayDriver<Query>, SqlQueryDisplayDriver>();
-            services.AddScoped<IQuerySource, SqlQuerySource>();
-            services.AddScoped<INavigationProvider, AdminMenu>();
+            services.AddScoped<SqlQuerySource>();
+            services.AddScoped<IQuerySource>(sp => sp.GetService<SqlQuerySource>());
+            services.AddKeyedScoped<IQuerySource>(SqlQuerySource.SourceName, (sp, key) => sp.GetService<SqlQuerySource>());
```

Comment:
Maybe we should add an extension method for registering an `IQuerySource`, it's already repeated several times here.

Reply:
done